lib/mocha.js

```diff
@@ -95,29 +96,47 @@ function Mocha(options) {
   options = options || {};
   this.files = [];
   this.options = options;
-  if (options.grep) {
-    this.grep(new RegExp(options.grep));
-  }
-  if (options.fgrep) {
-    this.fgrep(options.fgrep);
-  }
+  // root suite
   this.suite = new exports.Suite('', new exports.Context());
-  this.ui(options.ui);
-  this.bail(options.bail);
-  this.reporter(options.reporter, options.reporterOptions);
-  if (typeof options.timeout !== 'undefined' && options.timeout !== null) {
-    this.timeout(options.timeout);
+
+  this.grep(options.grep)
+    .fgrep(options.fgrep)
+    .ui(options.ui || 'bdd')
+    .bail(options.bail)
+    .reporter(options.reporter, options.reporterOptions)
+    .useColors(options.useColors)
+    .timeout(options.timeout || 200)
```

Comment:
Isn't the `timeout` default value part of the `yargs` setup already.

Reply:
this would assume the only thing invoking the constructor is Yargs...

Reply:
...but that suggests the default values should exist in a single place, and whatever needs them should pull them out.

A "default" JSON config file would make sense, IMO; we'd pull this in when constructed, and it could be inlined when bundling for the browser.